http/lbtransport/policy_test.go

```diff
@@ -40,75 +37,70 @@ func TestRoundRobinPolicy_PickWithBlacklist(t *testing.T) {
 		},
 	}
 
+	// Test Global blacklist.
+	picker := rr.Picker()
 	// Picking serially should give targets in exact order.
-	target, err := rr.Pick(req, testTargets)
+	target, err := picker.Pick(req, testTargets)
 	require.NoError(t, err)
 	assert.Equal(t, testTargets[1], target)
 
-	target, err = rr.Pick(req, testTargets)
+	target, err = picker.Pick(req, testTargets)
 	require.NoError(t, err)
 	assert.Equal(t, testTargets[2], target)
 
-	target, err = rr.Pick(req, testTargets)
+	target, err = picker.Pick(req, testTargets)
 	require.NoError(t, err)
 	assert.Equal(t, testTargets[0], target)
 
-	target, err = rr.Pick(req, testTargets)
+	target, err = picker.Pick(req, testTargets)
 	require.NoError(t, err)
 	assert.Equal(t, testTargets[1], target)
 
 	rr.atomicCounter = 0
-	rr.tryDialFunc = func(_ context.Context, target *Target) bool {
-		if target == testTargets[0] {
-			return false
-		}
-		return true
-	}
+	picker.ExcludeTarget(testTargets[0])
+
+	// To test global blacklist we need to spawn another picker!
+	picker = rr.Picker()
```

Comment:
is it better to make a picker with a different name? Otherwise it is not clear that these are two separate pickers